integration-tests/run-local-tests.sh

```diff
@@ -42,9 +42,11 @@ cd ${INTEGRATION_TESTS_DIR}
 
 set -x # activate printing executed commands
 
-mvn clean verify $ARGS \
+mvn clean verify $ARGS  \
   -Pcucumber \
   -Devent-bridge.manager.url=${MANAGER_URL} \
-  -Dkeycloak.realm.url=${KEYCLOAK_URL}/auth/realms/event-bridge-fm
-  # -Dbridge.token.username=${} \
-  # -Dbridge.token.password=${}
\ No newline at end of file
+  -Dkeycloak.realm.url=${KEYCLOAK_URL}/auth/realms/event-bridge-fm \
+  -Dbridge.token.username=kermit \
+  -Dbridge.token.password=thefrog \
+  -Dbridge.client.id=event-bridge \
+  -Dbridge.client.secret=secret
```

Comment:
we have a service account also for our local keycloak, so you don't really have to support user's credentials and service accounts https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/de7a969956da1b7286df30262f2369b57e028af0/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json#L329

Reply:
Sorry, I am not familiar with that. Could I ask for an example how to use it instead of system property? Thanks

Reply:
I mean you are doing 
```
        if (!USER_NAME.isEmpty() && !PASSWORD.isEmpty()) {
            return given().param("grant_type", "password")
                    .param("username", USER_NAME)
                    .param("password", PASSWORD)
                    .param("client_id", CLIENT_ID)
                    .param("client_secret", CLIENT_SECRET)
                    .contentType("application/x-www-form-urlencoded")
                    .accept(ContentType.JSON)
                    .when()
                    .post(keycloakURL + "/protocol/openid-connect/token").as(AccessTokenResponse.class)
                    .getToken();
        }

        return given().param("grant_type", "client_credentials")
                   .blablabla();
 ```
 
but you can simplify and use just the code path of service account, because we have them also for our local instance

Reply:
Thanks, I did so.